platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/state/FakeConsensusStateEventHandler.java

```diff
@@ -69,13 +69,15 @@ public enum FakeConsensusStateEventHandler implements ConsensusStateEventHandler
             .withConfigDataType(FileSystemManagerConfig.class)
             .build();
 
+    public List<VirtualMap> virtualMaps = new ArrayList<>();
```

Comment:
I am not sure how this is used, but on the surface, this looks wrong. `FAKE_CONSENSUS_STATE_EVENT_HANDLER` is a Singleton. Being able to store state and to close it is surprising.

Reply:
Refactored